Workbooks/ESETPROTECT.json

```diff
@@ -0,0 +1,233 @@
+{
+    "version": "Notebook/1.0",
+    "items": [
+      {
+        "type": 1,
+        "content": {
+          "json": "## ESET PROTECT"
```

Comment:
Should include note about parser, which is included in solution. See https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/Box/Workbooks/Box.json#L7 for example.